repo_tests/slm_frontend_shell_publish_test.py

```
@@ -0,0 +1,310 @@
+the detectors match nothing.
+
+Lives in ``repo_tests/`` for the same reason its Ansible-side siblings do:
+CI's shard command passes an explicit path list and neither
+``autobot-infrastructure/`` nor the repo root script is on it.
+"""
+
+from __future__ import annotations
+
+import re
+from pathlib import Path
+from typing import Iterator
```

Ruff (0.16.3)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)